upup/pkg/fi/has_address.go

```diff
@@ -19,9 +19,9 @@ package fi
 // HasAddress is implemented by elastic/floating IP addresses in order to include
 // relevant dynamically allocated addresses in the api-server's server TLS certificate.
 type HasAddress interface {
-	Task
+	//Task
```

Comment:
`CloudTask`? I don't want anything not a `Task` to implement this, as such things won't have `FindAddresses()` called.